File: PrefrenceSettingActivity/app/src/main/java/com/prefrence/prefrencesettingactivity/UserSettingActivity.java
package com.prefrence.prefrencesettingactivity;

import android.app.Activity;
import android.os.Bundle;

 
public class UserSettingActivity extends Activity {
 

	@Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState); 
        //this makes it incompatible with API 10:
        getFragmentManager().beginTransaction().replace(android.R.id.content,
                new SettingFragment()).commit();
 
    }
}

File: kotlin/fix/PrefrenceSettingActivity/app/src/main/java/com/dstrube/prefrencesettingactivity/UserSettingActivity.kt
package com.dstrube.prefrencesettingactivity

import android.os.Bundle
import android.app.Activity


class UserSettingActivity : Activity() {


    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //this makes it incompatible with API 10:
        fragmentManager.beginTransaction().replace(android.R.id.content,
                SettingFragment()).commit()

    }
}
